src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -203,32 +218,37 @@ const formConfig = {
         disabilitiesOrientation: {
           title: '',
           path: 'disabilities/orientation',
+          // Only show the page if both (or potentially neither) options are chosen on the claim-type page
```

Comment:
Is this possible though? The claimsType page uses `validateBooleanGroup` to ensure at least one selection is made (https://github.com/department-of-veterans-affairs/vets-website/pull/9470/files#diff-1ac68cd612027c8f9af3b38cad4b7633R14)

Reply:
Is the "or potentially neither" possible? No, not right now. This was basically just a safeguard against whether we decide to make that question not required either as a part of this PR (which we didn't) or in the future.

When I originally wrote this, we weren't clear on whether the question should be required, so I made it work either way. Later, when we decided to require the claim type question, I decided to keep this around in case we change our minds later.

Reply:
Well that seems like a good idea 💯